lib/formatters/codeframe.js

```diff
@@ -63,7 +64,9 @@ function formatMessage(message, parentResult) {
 
     if (sourceCode) {
         result.push(
-            codeFrame(sourceCode, message.line, message.column, { highlightCode: false })
+            codeFrame.codeFrameColumns(sourceCode, {
+                start: { line: message.line, column: message.column }
+            }, { highlightCode: true })
```

Comment:
Based on https://github.com/eslint/eslint/pull/7437#pullrequestreview-5370538 it looks like this formatter intentionally disables `highlightCode` for performance reasons. Do you know if there have been any performance improvements with the `highlightCode` option that would make it worthwhile now?

Reply:
I'm not aware. I haven't experienced performance issues with it.